app/src/main/res/layout/activity_barrier_free_contact_listview.xml

```diff
@@ -48,6 +48,7 @@
         android:textSize="@dimen/semilarge_text_size"
         android:visibility="gone"
         tools:text="Person More info"
+        android:backgroundTint="@color/calendar_amber"
```

Comment:
What does this tint do?
It does not appear to be amber on my screen.
would `@color/text_secondary` be a better fit?

![image](https://github.com/TUM-Dev/Campus-Android/assets/26258709/f624ae8d-c7e1-43d0-a2cb-2da5874d6c0c)

Reply:
Yes, that's not the right color. It's been a while since I've changed that and now I don't know why anymore